### PR TITLE
Navbar and footer addition

### PR DESCRIPTION
**Pull request by mikolaslorant**
## Summary
-Added navbar 
-Added footer
-Added logos

## Screenshot

Navbar screenshot:
navbar

Footer screenshot:
footer

## Trello cards
-Footer
-Navbar

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,80 +6,90 @@
...
-      <div class="tech">
-        <img class="tech-logo" src="assets/images/react_icon.svg" alt="React logo"/>
-        <h1 class="tech-title">React JS</h1>
-        <strong class="tech-text">Release: 2013</strong>
-        <p class="tech-text">React is a declarative, component-based
-          JavaScript library for building user interfaces.</p>
+  <body>
+    <!--Navbar-->
+    <nav class="navbar">
+      <img class="wlogo" src="assets/images/wolox_logo.svg"/>
```
Comment (FrankIglesias) [inline index.html]: add an alt

File: index.html [hunk 2]
```diff
@@ -6,80 +6,90 @@
...
+      <p class="developer-tools">Developer Tools</p>
+    </nav>
+    <div class="center-content">
+      <!-- Tech set front-end-->
+      <h1 class="title">Front-End Tools</h1>
+      <div class="tech-set">
+        <div class="tech">
+          <img class="tech-logo" src="assets/images/react_icon.svg" alt="React logo"/>
```
Comment (FrankIglesias) [inline index.html]: space before slash

File: index.html [hunk 3]
```diff
@@ -6,80 +6,90 @@
...
+          <h1 class="tech-title">React JS</h1>
+          <strong class="tech-text">Release: 2013</strong>
+          <p class="tech-text">React is a declarative, component-based
+            JavaScript library for building user interfaces.</p>
+        </div>
+        <div class="tech">
+          <img class="tech-logo" src="assets/images/angular_icon.svg" alt="Angular logo"/>
```
Comment (FrankIglesias) [inline index.html]: same

File: scss/navbar.scss [hunk 4]
```diff
@@ -0,0 +1,19 @@
+.navbar {
+  background-color: white;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: use variable color

File: scss/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+}
+
+.developer-tools {
+  float:right;
+  font-size: $text-big;
+  margin: 20px 20px auto 10px;
+}
+
+.wlogo {
+  margin: 20px 10px auto 10px;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: margin 20px auto 10px;

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by mikolaslorant** [comment]
Corrected requested changes

File: scss/footer.scss [hunk 6]
```diff
@@ -0,0 +1,21 @@
...
+}
+
+.footer-logo {
+  display:block;
+  height: 40px;
+  margin: 15px auto;
+}
+
+.slogan {
+  color: white;
```
Comment (FrankIglesias) [inline scss/footer.scss]: use a variable color

File: scss/navbar.scss [hunk 7]
```diff
@@ -0,0 +1,22 @@
+.navbar {
+  background-color: $white;
+  height: 70px;
+  position:fixed;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: space after comma

File: scss/navbar.scss [hunk 8]
```diff
@@ -0,0 +1,22 @@
...
+  width: 100%;
+  z-index: 1;
+  -webkit-box-shadow: 0px 6px 27px -7px $box-shadow-color;
+  -moz-box-shadow: 0px 6px 27px -7px $box-shadow-color;
+  box-shadow: 0px 6px 27px -7px $box-shadow-color;
+}
+
+.developer-tools {
+  float:right;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: same

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Comment by mikolaslorant** [comment]
Corrected new requested changes

File: scss/navbar.scss [hunk 9]
```diff
@@ -0,0 +1,22 @@
...
+  position: fixed;
+  width: 100%;
+  z-index: 1;
+  -webkit-box-shadow: 0px 6px 27px -7px $box-shadow-color;
+  -moz-box-shadow: 0px 6px 27px -7px $box-shadow-color;
+  box-shadow: 0px 6px 27px -7px $box-shadow-color;
+}
+
+.developer-tools {
+  float: right;
```
Comment (FrankIglesias) [inline scss/navbar.scss]: mmmmm you can apply a flexbox property to the navbar to avoid this

**Review by FrankIglesias — CHANGES_REQUESTED** [review]

**Review by FrankIglesias — APPROVED** [review]

**Pull request merged**